### PR TITLE
Release: v1.33.0

### PR DESCRIPTION
**Pull request by spowelljr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+* Add active kubecontext to `minikube profile list` output [#17735](https://github.com/kubernetes/minikube/pull/17735)
+* CNI calico: support kubeadm.pod-network-cidr [#18233](https://github.com/kubernetes/minikube/pull/18233)
+* CNI bridge: Ensure pod communications are allowed [#16143](https://github.com/kubernetes/minikube/pull/16143)
+
+Bugs:
+* Fix unescaped local host regex [#18617](https://github.com/kubernetes/minikube/pull/18617)
+* Fix regex on validateNetwork to support special characters [#18158](https://github.com/kubernetes/minikube/pull/18158)
+
+Version Upgrades:
+* Bump Kubernetes version default: v1.30.0 and latest: v1.30.0 [#18669](https://github.com/kubernetes/minikube/pull/18669)
```
Comment (medyagh) [inline CHANGELOG.md:18]: we could mention this in the features

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,74 @@
 # Release Notes
 
+## Version 1.33.0 - 2024-04-19
+
+Features:
```
Comment (medyagh) [inline CHANGELOG.md:5]: lets include beta changes into this one too

**Review by medyagh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (medyagh) [reply CHANGELOG.md:5]: nevermind we put the merge changes of beta and stable into the release note

File: CHANGELOG.md (hunk 1 above)
Comment (spowelljr) [reply CHANGELOG.md:19]: Done

**Review by medyagh — APPROVED** [review]

**Review by medyagh — APPROVED** [review]

**Pull request merged**